features/communities/events/AttendanceMenu.tsx

```diff
@@ -17,12 +17,12 @@ const useStyles = makeStyles((theme) => ({
 
 export default function AttendanceMenu({
   loading,
-  onChange,
+  onChangeAttendanceState,
   attendanceState,
   id,
 }: {
   loading: boolean;
-  onChange: (attendanceState: AttendanceState) => void;
+  onChangeAttendanceState: (attendanceState: AttendanceState) => void;
```

Comment:
Personally I prefer the short version for these variable names (here and below), since there's already the type information when you hover telling you it is about `AttendanceState`, so adding it to the variable name is doubling up. (Unless there's something I haven't considered?)

Reply:
The namings are not clear enough for me as a first-time viewer. and I would have to go back to the props to check what is it `onChange` about. What do you think?